cmd/status/printers/event/event_printer.go

```diff
@@ -44,8 +44,12 @@ func (ep *eventPrinter) Print(ch <-chan pollevent.Event, identifiers []object.Ob
 			cancelFunc(statusCollector, e)
 		}),
 	)
-	// Block until the done channel is closed.
-	<-done
+	// Listen to the channel until it is closed.
+	var err error
+	for msg := range done {
+		err = msg.Err
+	}
```

Comment:
Is there a chance that it can return multiple items? If yes, do we want to return the first one? If no, everything looks good to me.

Reply:
No, it can only return one. An error event means the status poller is closing down.